### PR TITLE
Make Measure a Metric and add support for Handle.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Start implementing OTEPS-0003:

* Add Handle support for Measure
* Remove component from Builder that will be part of getting a Meter
* Remove for the moment the code copied from OpenCensus related to Views.

### CODE REVIEW

File: api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java [hunk 1]
```diff
@@ -453,27 +429,61 @@ public CounterLong build() {
...
+    public void setCallback(Runnable metricUpdater) {
+      Utils.checkNotNull(metricUpdater, "metricUpdater");
+    }
+
+    @Override
+    public void removeHandle(List<String> labelValues) {
+      Utils.checkNotNull(labelValues, "labelValues");
+    }
+
+    private static final class NoopHandle implements Handle {
```
Comment (songy23) [inline api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java]: `@ThreadSafe`?

File: api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java [hunk 2]
```diff
@@ -486,63 +496,112 @@ public Builder setUnit(String unit) {
...
```
Comment (songy23) [inline api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java]: Same.

File: api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java (hunk 1 above)
Comment (bogdandrutu) [reply api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java]: Done.

File: api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java (hunk 2 above)
Comment (bogdandrutu) [reply api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java]: Done.

**Comment by bogdandrutu** [comment]
I have a bit of a problem with View/ViewManager because Measure uses generic, I will think about a solution later and bring the View class back. I think the ViewManager functionality will probably be in the MeterSdk or a different component.

**Pull request merged**